code_route/cli/app.py

```
@@ -0,0 +1,1303 @@
+        if session_info:
+            msg_count = session_info.message_count
+            if msg_count > 0:
+                self.console.print(
+                    f"[dim]Resumed session with {msg_count} messages[/dim]"
+                )
+
+                # Show last few messages as context
+                if self.session_manager._messages_cache:
+                    recent = self.session_manager._messages_cache[-4:]
+                    for msg in recent:
+                        role = "You" if msg.role.value == "user" else "Assistant"
```

Ruff (0.14.14)

[error] Local variable `role` is assigned to but never used

Remove assignment to unused variable `role`

(F841)